generators/entity-client/templates/react/src/main/webapp/app/entities/entity.tsx.ejs

```diff
@@ -193,7 +197,6 @@ export class <%= entityReactName %> extends React.Component<I<%= entityReactName
           <InfiniteScroll pageStart={this.state.activePage}
                           loadMore={this.handleLoadMore}
                           hasMore={this.state.activePage <= this.props.links.last}
-                          loader={<div className="loader">Loading ...</div>}
```

Comment:
Actually, it is nice to show some indication that things are loading, why did you remove this?